app/assets/javascripts/short-form/directives/communityScreeningHeader.js.coffee

```diff
@@ -0,0 +1,27 @@
+angular.module('dahlia.directives')
+.directive 'communityScreeningHeader', ['$translate', 'ListingService', ($translate, ListingService) ->
+  replace: true
+  scope: true
+  templateUrl: 'short-form/directives/community-screening-header.html'
+
+  link: (scope, elem, attrs) ->
+    listing = ListingService.listing
+    restriction = listing.STUB_CommunityRestriction
+    age = { minAge: listing.Reserved_community_minimum_age }
```

Comment:
My intuition might have been to pass these in as attributes rather than get from Listing service--would love to discuss to best practices you've learned for directives at some point.